<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_PartNo">
       <property name="styleSheet">
        <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
background-color: rgba(0, 0, 0, 80);
border: 3px solid;
border-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#c8c8c8;&quot;&gt;5QM121251R&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_tittle">
       <property name="styleSheet">
        <string notr="true">color: rgb(200, 200, 200);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Numero de Parte:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Box_PartNo">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(44, 49, 58);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btn_Save">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgba(0, 85, 127, 150);
font:18pt &quot;Segoe UI&quot;;
color: rgb(200, 200, 200);
border-radius: 4px;
border: 2px solid black;
border-color: rgb(0, 85, 127);}

QPushButton:pressed {
	background-color: rgb(0, 85, 127);
}
</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Cancel">
         <property name="styleSheet">
          <string notr="true">QPushButton{background-color: rgba(207, 162, 9,150);
font:18pt &quot;Segoe UI&quot;;
color: rgb(200, 200, 200);
border-radius: 4px;
border: 2px solid black;
border-color: rgb(207, 162, 9);}

QPushButton:pressed {
	background-color: rgb(255, 233, 51 );
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
